lib/plotting_functions.py

```diff
@@ -123,6 +133,34 @@
 #HELPER FUNCTIONS
 #################
 
+def load_dataset(fils):
+    """
+    This method exists to get an xarray Dataset from input file information that can be passed into the plotting methods.
+
+    Parameters
+    ----------
+    fils : list
+        strings or paths to input file(s)
+
+    Returns
+    -------
+    xr.Dataset
+
+    Notes
+    -----
+    When just one entry is provided, use `open_dataset`, otherwise `open_mfdatset`
+    """
+    if len(fils) == 0:
+        warnings.warn(f"Input file list is empty.")
+        return None
+    elif len(fils) > 1:
+        return xr.open_mfdataset(fils, combine='by_coords')
+    else:
+        sfil = str(fils[0])
+        return xr.open_dataset(sfil)
```

Comment:
Does `fils` have to be converted to a string here?  I thought the list was already assumed to contain either strings or path objects?  If not then we can simplify these lines to be:

 ```suggestion
        return xr.open_dataset(fils[0])
```